code/go/0chain.net/blobbercore/handler/handler_middlewares.go

```diff
@@ -27,8 +20,8 @@ func useCors(h http.Handler) http.Handler {
 			}
 		}()
 
-		w.Header().Add("Access-Control-Allow-Headers", strings.Join(allowedHeaders, ", "))
-		w.Header().Add("Access-Control-Allow-Origin", strings.Join(allowedOrigins, ", "))
+		w.Header().Add("Access-Control-Allow-Headers", "*")
```

Comment:
Should we change from limited headers to allow all?

Reply:
yes. It is unnecessary to limit some headers for it